cumulus/pallets/xcmp-queue/src/mock.rs

```diff
@@ -55,6 +55,7 @@ parameter_types! {
 
 type AccountId = u64;
 
+#[derive_impl(frame_system::config_preludes::TestDefaultConfig as frame_system::DefaultConfig)]
```

Comment:
We should really work on this syntax and make it better.

Reply:
Yes, will add this to the tracking issue